test/test_linked_list.c

```diff
@@ -0,0 +1,82 @@
+#include <unity.h>
+#include "simulation_lifecycle/utils/linked_list.h"
+
+void setUp(void) {
```

Comment:
set_up

Reply:
The `setUp` and `tearDown` functions are part of Unity, we cannot rename them. Otherwise, the program won't compile.